sonar-orchestrator/src/main/java/com/sonar/orchestrator/config/Licenses.java

```diff
@@ -52,30 +51,26 @@ public Licenses(Configuration configuration) {
   }
 
   public String getLicense(Edition edition, Version version) {
-    if (version.isGreaterThanOrEquals(7,2)) {
-      return licensesPerEdition.computeIfAbsent(edition, e -> {
-        String filename;
-        switch (e) {
-          case DEVELOPER:
-            filename = "de.txt";
-            break;
-          case ENTERPRISE:
-            filename = "ee.txt";
-            break;
-          case DATACENTER:
-            filename = "dce.txt";
-            break;
-          default:
-            throw new IllegalStateException("License does not exist for edition " + e);
-        }
-        return download(baseUrl + "master/edition_testing/" + filename);
-      });
+    if (!version.isGreaterThanOrEquals(7, 9)) {
```

Comment:
You can replace these 3 lines by only one line : 
```
Preconditions.checkArgument(server.version().isGreaterThanOrEquals(7, 9), "Commercial licenses of SonarQube %s are no longer supported", server.version()));
```

Reply:
In fact no, Guava is not used in Orchestrator!